README.md

```diff
@@ -67,3 +67,9 @@ This container delivers some metrics about the MySQL backups. The metrics are av
 | mysql_s3_backup_start_timestamp | gauge (timestamp in milliesconds) |
 | mysql_s3_backup_file_size_in_bytes{database="database", table="table"} | gauge (file size in bytes) |
 | mysql_s3_backup_duration_in_ms{database="database", table="table"} | gauge (duration in millieseconds) |
+
+## Grafana dashboard
+
+To use the preconfigured dashboard for Grafana you have to import the `grafana-dashboard/config.json`-file to your Grafana instance.
+
+![Grafana dashboard example](grafana-dashboard/example.png "Grafana dashboard example")
```

Comment:
I've just seen that you can also append `?raw=true` at the end of the path. This way the picture will be shown in full size when you click on it.
(see https://stackoverflow.com/questions/14494747/add-images-to-readme-md-on-github#answer-33006018)